### PR TITLE
[proj4] Update to version 6.3.1

### PR DESCRIPTION
**Pull request by orudge**
Updates proj4 to version 6.3.1

### CODE REVIEW

File: ports/proj4/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: proj4
-Version: 6.3.0-1
+Version: 6.3.1-1
```
Comment (PhoebeHui) [inline ports/proj4/CONTROL]: Could you update the version to '6.3.1' ?

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: ports/proj4/CONTROL (hunk 1 above)
Comment (orudge) [reply ports/proj4/CONTROL]: Have done so.

**Review by PhoebeHui — APPROVED** [review]

**Comment by strega-nil** [comment]
Thanks @orudge :)

**Pull request merged**